torchstain/torch/normalizers/macenko.py

```diff
@@ -49,7 +49,7 @@ def __find_concentration(self, OD, HE):
         Y = OD.T
 
         # determine concentrations of the individual stains
```

Comment:
Note that I am not sure that doing this is backward compatible, as I believe `torch.linalg.lstsq` does not exist in older versions of torch. You will need to find a fix that works for all versions.

Reply:
By the Torch repo that change was made in Torch 1.9. I suggest bumping the minimum version of Torch required from 1.8 to 1.9 to assure backward compatibility:

https://github.com/pytorch/pytorch/blob/b0c86caa1d46a16195682e2afe5456f97265aa53/torch/_linalg_utils.py#L111